tests/test_device.py

```diff
@@ -297,6 +297,8 @@ def test_sync_execute_register_no_hid_tag_not_linked(tag_id: str):
     """
     tag = Tag(id=tag_id)
     pc = d.Desktop(**conftest.file('pc-components.db')['device'], tags=OrderedSet([tag]))
+    from flask import g
```

Comment:
May import statement be moved to the head of the file?

Reply:
Yes, I understand that it is better to always import globally. I'll change it.